webrtc.html

```diff
@@ -481,8 +481,9 @@
           </li>
 
           <li>
-            <p>Initialize an internal variable to represent a queue of
-            <code>operations</code> with an empty set.</p>
+            <p>On the <var>connection</var>, reserve an internal slot
+            <var>operations</var>, and initialize an internal slot
+            <var>operationsLength</var> to <code>0</code>.</p>
           </li>
```

Comment:
I still prefer the text speaking in terms of "queue of operations" rather than "reserve an internal slot".  Can you not keep the text in terms of "queue of operations" and also fix the deficiencies you are trying to address?

You argument seems to be "X has deficiencies, so we should replace it with Y", when just tweaking X a little bit might address the deficiencies without turning it into Y.